extensions/standard-processors/processors/TailFile.cpp

```diff
@@ -135,6 +135,13 @@ const core::Property TailFile::AttributeProviderService(
         ->asType<minifi::controllers::AttributeProviderService>()
         ->build());
 
+const core::Property TailFile::BatchSize(
+    core::PropertyBuilder::createProperty("Batch Size")
+        ->withDescription("Maximum number of lines to process in a single trigger. If set to 0 all new lines will be processed.")
```

Comment:
should we change this to "number of flowfiles emitted" or similar? with the input delimiter the user could break the "flowfile == line" correspondence

Reply:
Good point, updated in 24f9ab5c32083fea259b4b1e0889bd93f20e8cda